lib/widgets/draggable_scrollable_modal_bottom_sheet.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+
+class _DraggableScrollableLayer extends StatelessWidget {
+  const _DraggableScrollableLayer({required this.builder});
+
+  final WidgetBuilder builder;
+
+  @override
+  Widget build(BuildContext context) {
+    return DraggableScrollableSheet(
+        minChildSize: 0.25,
+        initialChildSize: 0.26,
```

Comment:
Let's leave a comment pointing to https://github.com/zulip/zulip-flutter/pull/12#discussion_r1116423455 for why we're setting one of these to epsilon more than the other.